### PR TITLE
Implement COM Betrag

### LINKED ISSUE #144: Implement COM Betrag
https://www.bo4e.de/dokumentation/komponenten/com-betrag
requires 
- #107

### PR DESCRIPTION
**Pull request by hf-kklein**
Fixes #144

### CODE REVIEW

File: src/bo4e/com/betrag.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+# pylint: disable=too-few-public-methods
+from bo4e.enum.waehrungscode import Waehrungscode
+
+
+@attr.s(auto_attribs=True, kw_only=True)
+class Betrag(COM):
+    """
+    Die Komponente wird dazu verwendet, Summebeträge (beispielsweise in Angeboten und Rechnungen) als Geldbeträge a
+    bzubilden. Die Einheit ist dabei immer die Hauptwährung also Euro, Dollar etc…
```
Comment (hf-krechan) [inline src/bo4e/com/betrag.py:22]:
```suggestion
    Die Komponente wird dazu verwendet, Summenbeträge (beispielsweise in Angeboten und Rechnungen) als Geldbeträge abzubilden.
    Die Einheit ist dabei immer die Hauptwährung also Euro, Dollar etc…
```

File: src/bo4e/com/betrag.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    """
+
+    # required attributes
+    wert: Decimal = attr.ib(validator=attr.validators.instance_of(Decimal))  #: Gibt den Betrag des Preises an.
+    waehrung: Waehrungscode = attr.ib(
```
Comment (hf-krechan) [inline src/bo4e/com/betrag.py:27]: The BO4E docu writes `Waehrungs**C**ode` ?

File: src/bo4e/com/betrag.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    )  #: Die entsprechende Waehrung
+
+
+class BetragSchema(COMSchema):
+    """
+    Schema for de-/serialization of Betrag.
+    """
+
+    # required attributes
+    wert = fields.Decimal(as_string=True, places=6)
```
Comment (hf-krechan) [inline src/bo4e/com/betrag.py:38]: Where is this `places=6` defined?

**Review by hf-krechan — CHANGES_REQUESTED** [review]

File: src/bo4e/com/betrag.py (hunk 3 above)
Comment (hf-kklein) [reply src/bo4e/com/betrag.py:38]: nirgends. ich schau mal, ob ich was drum rum gebaut kriege.

File: src/bo4e/com/betrag.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+# pylint: disable=too-few-public-methods
+from bo4e.enum.waehrungscode import Waehrungscode
+
+
+@attr.s(auto_attribs=True, kw_only=True)
+class Betrag(COM):
+    """
+    Die Komponente wird dazu verwendet, Summenbeträge (beispielsweise in Angeboten und Rechnungen) als Geldbeträge abzubilden.
+    Die Einheit ist dabei immer die Hauptwährung also Euro, Dollar etc…
```
Comment (hf-krechan) [inline src/bo4e/com/betrag.py:22]: ♥

File: src/bo4e/com/betrag.py (hunk 2 above)
Comment (hf-kklein) [reply src/bo4e/com/betrag.py:27]: #236

File: src/bo4e/com/betrag.py (hunk 3 above)
Comment (hf-kklein) [reply src/bo4e/com/betrag.py]: https://github.com/Hochfrequenz/BO4E-python/pull/231/commits/373150c16c9f47ebf6ba0a08c02556d3103b28d0

**Review by hf-krechan — APPROVED** [review]

**Pull request merged**